plat/common/include/x86/paging.h

```diff
@@ -98,13 +99,15 @@ pgarch_pte_create(__paddr_t paddr, unsigned long attr, unsigned int level,
 	if (!(attr & PAGE_ATTR_PROT_EXEC))
 		pte |= X86_PTE_NX;
 
+	if (attr & PAGE_ATTR_WRITECOMBINE) {
+		pte |= X86_PTE_PCD;
+		pte |= X86_PTE_PWT;
+	}
+
 	/* Take all other bits from template */
 	pte |= template & (X86_PTE_US |
-			   X86_PTE_PWT |
-			   X86_PTE_PCD |
```

Comment:
We should also remove these from the `pt_pte_create` variant then. Make sure to add a remark in the comment similar to the one for the protections.